renderer/index.css

```diff
@@ -50,19 +50,22 @@ table {
 }
 
 @keyframes fadein {
-  from { opacity: 0; }
-  to   { opacity: 1; }
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
 }
 
-
 .app {
   -webkit-user-select: none;
   -webkit-app-region: drag;
   height: 100%;
   display: flex;
   flex-flow: column;
-  animation: fadein 0.3s;
   background: rgb(40, 40, 40);
+  animation: fadein 1s;
```

Comment:
just tested it. feels v nice! in fact it feels smoother and more professional than other electron apps now

* Atom: pops up a gray page, then a half second later gray text appears, then a half second later the syntax highlighting activates

* Spotify: pops up a gray page, then some HTML, redoes the layout a couple times, janks around, finally settles down a few seconds later

* Webtorrent Desktop: pops up a gray page, fades in the UI 👍